acbuild/copy.go

```diff
@@ -20,9 +20,9 @@ import (
 
 var (
 	cmdCopy = &cobra.Command{
-		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
+		Use:     "copy PATH_ON_HOST... PATH_IN_ACI",
```

Comment:
I guess really we have two modes now like `cp`:
```
$ cp --help
Usage: cp [OPTION]... [-T] SOURCE DEST
  or:  cp [OPTION]... SOURCE... DIRECTORY
...
``` 
so really it should be more like `copy PATH_ON_HOST PATH_IN_ACI` and `copy PATH_ON_HOST... DIRECTORY_IN_ACI` - don't suppose we can have multilines here too?